diff -urN teclibs/cd/tecmake.mak teclibs_new/cd/tecmake.mak
--- teclibs/cd/tecmake.mak	2016-09-15 04:43:10.000000000 +1000
+++ teclibs_new/cd/tecmake.mak	2018-01-08 10:37:51.000000000 +1100
@@ -565,7 +565,11 @@
   #Homebrew
   #FREETYPE_INC := /usr/local/include/freetype2
   #Fink
-  FREETYPE_INC := /sw/include/freetype2
+  #FREETYPE_INC := /sw/include/freetype2
+  #MacPorts
+  #FREETYPE_INC := /opt/local/include/freetype2
+  #Custom
+  FREETYPE_INC := $(HOME)/CHDK/teclibs/freetype/include
 endif
 
 # Definitions for GTK
@@ -726,9 +730,12 @@
   #STDINCS += /usr/local/include
   #LDIR += /usr/local/lib
   #Fink
-  STDINCS += /sw/include
-  LDIR += /sw/lib
-  
+  #STDINCS += /sw/include
+  #LDIR += /sw/lib
+  #Macports
+  STDINCS += /opt/local/include
+  LDIR += /opt/local/lib
+
   UNIX_BSD = Yes
   X11_LIBS := Xp Xext X11
   X11_LIB := /usr/X11R6/lib /usr/X11/lib
@@ -736,7 +743,7 @@
   MOTIF_INC := /usr/OpenMotif/include
   MOTIF_LIB := /usr/OpenMotif/lib
   ifdef BUILD_DYLIB
-    STDLDFLAGS := -dynamiclib -Wl -fno-common -headerpad_max_install_names -undefined dynamic_lookup -install_name lib$(TARGETNAME).dylib
+    STDLDFLAGS := -dynamiclib -fno-common -headerpad_max_install_names -undefined dynamic_lookup -install_name lib$(TARGETNAME).dylib
     DLIBEXT := dylib
     STDFLAGS += -fno-common
   else
diff -urN teclibs/freetype/tecmake.mak teclibs_new/freetype/tecmake.mak
--- teclibs/freetype/tecmake.mak	2016-09-15 04:43:10.000000000 +1000
+++ teclibs_new/freetype/tecmake.mak	2018-01-08 10:03:55.000000000 +1100
@@ -736,7 +736,7 @@
   MOTIF_INC := /usr/OpenMotif/include
   MOTIF_LIB := /usr/OpenMotif/lib
   ifdef BUILD_DYLIB
-    STDLDFLAGS := -dynamiclib -Wl -fno-common -headerpad_max_install_names -undefined dynamic_lookup -install_name lib$(TARGETNAME).dylib
+    STDLDFLAGS := -dynamiclib -fno-common -headerpad_max_install_names -undefined dynamic_lookup -install_name lib$(TARGETNAME).dylib
     DLIBEXT := dylib
     STDFLAGS += -fno-common
   else
diff -urN teclibs/im/tecmake.mak teclibs_new/im/tecmake.mak
--- teclibs/im/tecmake.mak	2016-09-15 04:43:10.000000000 +1000
+++ teclibs_new/im/tecmake.mak	2018-01-08 10:05:40.000000000 +1100
@@ -736,7 +736,7 @@
   MOTIF_INC := /usr/OpenMotif/include
   MOTIF_LIB := /usr/OpenMotif/lib
   ifdef BUILD_DYLIB
-    STDLDFLAGS := -dynamiclib -Wl -fno-common -headerpad_max_install_names -undefined dynamic_lookup -install_name lib$(TARGETNAME).dylib
+    STDLDFLAGS := -dynamiclib -fno-common -headerpad_max_install_names -undefined dynamic_lookup -install_name lib$(TARGETNAME).dylib
     DLIBEXT := dylib
     STDFLAGS += -fno-common
   else
diff -urN teclibs/iup/tecmake.mak teclibs_new/iup/tecmake.mak
--- teclibs/iup/tecmake.mak	2016-11-15 08:52:55.000000000 +1100
+++ teclibs_new/iup/tecmake.mak	2018-01-08 11:08:53.000000000 +1100
@@ -726,8 +726,11 @@
   #STDINCS += /usr/local/include
   #LDIR += /usr/local/lib
   #Fink
-  STDINCS += /sw/include
-  LDIR += /sw/lib
+  #STDINCS += /sw/include
+  #LDIR += /sw/lib
+  #Macports
+  STDINCS += /opt/local/include
+  LDIR += /opt/local/lib
   
   UNIX_BSD = Yes
   X11_LIBS := Xp Xext X11
@@ -736,7 +739,7 @@
   MOTIF_INC := /usr/OpenMotif/include
   MOTIF_LIB := /usr/OpenMotif/lib
   ifdef BUILD_DYLIB
-    STDLDFLAGS := -dynamiclib -Wl -fno-common -headerpad_max_install_names -undefined dynamic_lookup -install_name lib$(TARGETNAME).dylib
+    STDLDFLAGS := -dynamiclib -fno-common -headerpad_max_install_names -undefined dynamic_lookup -install_name lib$(TARGETNAME).dylib
     DLIBEXT := dylib
     STDFLAGS += -fno-common
   else
